src/wp-includes/blocks.php

```diff
@@ -175,11 +175,18 @@ function register_block_script_module_id( $metadata, $field_name, $index = 0 ) {
 	$block_version       = isset( $metadata['version'] ) ? $metadata['version'] : false;
 	$module_version      = isset( $module_asset['version'] ) ? $module_asset['version'] : $block_version;
 
+	$args = array();
+	if ( isset( $metadata['supports']['interactivity'] ) && $metadata['supports']['interactivity'] ) {
+		// TODO: Add ability for the fetchpriority to be specified in block.json for the viewScriptModule. In wp_default_script_modules() the fetchpriority defaults to low since server-side rendering is employed for core blocks, but there are no guarantees that this is the case for non-core blocks. That said, viewScriptModule entails Interactivity API, following the pattern from core it _should_ be SSR'ed and that is why this is the default for when the block supports interactivity.
```

Comment:
Maybe that property in `block.json` should become an object instead of a string. In the future we might need other properties in there too.

cc @gziolo

Reply:
Absolutely! See below 😄 https://github.com/WordPress/wordpress-develop/pull/8815#discussion_r2098332344